_drafts/SC4-1-2-button-has-name.md

```diff
@@ -0,0 +1,85 @@
+---
+name: Buttons have an accessible name
+description: |
+  Each button element has an accessible name
+
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects: # Remove what is not applicable
```

Comment:
Remove -> # Remove what is not applicable

Reply:
This is a comment (anything after hash #), and will not be shown or generated in the end result.
Either- way removed it.